tests/try/basic/test.sh

```diff
@@ -21,6 +21,17 @@ rlJournalStart
         rlAssertGrep "Run .* successfully finished. Bye for now!" $rlRun_LOG
```

Comment:
Can you assertnotgrep in this test or any other close one to make sure it is not applied in the non-contexed case?

Reply:
Added it to "Adjusted User Plan" to make it self-contained (reused ./verbose.exp).